pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -177,7 +177,7 @@ public SimpleLoadManagerImpl(PulsarService pulsar) {
         placementStrategy = new WRRPlacementStrategy();
         lastLoadReport = new LoadReport(pulsar.getWebServiceAddress(), pulsar.getWebServiceAddressTls(),
                 pulsar.getBrokerServiceUrl(), pulsar.getBrokerServiceUrlTls());
-        brokerHostUsage = new BrokerHostUsage(pulsar);
+        brokerHostUsage = new LinuxBrokerHostUsageImpl(pulsar);
```

Comment:
This will generate errors when running on MacOS. We should have a basic factory that returns a generic host usage implementation when not running on Linux.